harvey/images.py

```diff
@@ -2,66 +2,68 @@
 import subprocess
 
 import requests
+import docker
 
 from harvey.globals import Global
 
 
 class Image:
     @staticmethod
-    def build_image(config, webhook, context=''):
+    def build_image(config, webhook, context=""):
         """Build a Docker image by shelling out and running Docker commands."""
-        # TODO: Use the Docker API for building instead of a shell \
-        # command (haven't because I can't get it working)
         # tar = open('./docker/pullbug.tar.gz', encoding="latin-1").read()
         # json = open('./harvey/build.json', 'rb').read()
         # data = requests.post(Global.BASE_URL + 'build', \
         # params=json, data=tar, headers=Global.TAR_HEADERS)
```

Comment:
Let's remove these commented out lines since they were an attempt at a previous implementation.

Reply:
Ok, will make these changes in upcoming commits.